src/test/java/com/google/api/codegen/testdata/csharp_gapic_snippets_library.baseline

```diff
@@ -1,4 +1,4 @@
-============== file: Google/Example/Library/V1/LibraryServiceClientSnippets.g.cs ==============
+============== file: Google/Example/Library/V1/GeneratedLibraryServiceClientSnippets.g.cs ==============
```

Comment:
Can we keep the filename as LibraryServiceClientSnippets.g.cs, but change the class name?
That's the way I've got it now - it's a little odd, but it gives prominence to the manual code, while still making the generated code available easily. Not a problem if it's hard though.